assessment.md

```diff
@@ -239,17 +239,39 @@ TODO: add description for each assessment level for this skill.
 ### Interactions
 
 * Interaction with the customers
+- SERVICE
+- Social
+- Eager
+- Replace
+- Verbal Communication
+- Informative
+- Countability
+- Easy
 
-TODO: add description for each assessment level for this skill.
+The objective for the interaction with a customer is supposed to be smooth and easy. Interacting with customers is one of the most important parts of software engineering. The acronym of "SERVICE" is a great way for us to see what attributes and tendencies lead to good service.
 
 * Interaction with the team leaders
 
-TODO: add description for each assessment level for this skill.
+- **None**: refuses to interact with and/or work with team leader(s). Ignores advisement or assignment from team leader.
+- **Inadequate**: Interacts with and occasionally works with the team leader(s), but does not actively participate in completing tasks assigned by the team leader(s).
+- **Adequate**: Interacts with and works with the team leader(s) while completing most of the tasks assigned to them by the team leader(s).
+- **Good**: Interacts with, works with, and actively collaborates with the team leader(s). Completes all tasks assigned to them by the team leader(s).
+- **Excellent**: Interacts with, works with, and actively collaborates with the team leader(s). Completes all tasks assigned to them by the team leader(s) and offers additional help to expedite the completion of other related tasks to complete the project.
 
 * Interaction with the course instructor
 
-TODO: add description for each assessment level for this skill.
+- Effective interaction with the course instructor would be being respectful. The both student and instructor should respect each other's time and effort. Students should have appropriate specific questions about programming/ technical problems. Students have to let the instructor know if he/she is understanding the topics and keeping up with her.
+
+- Ineffective interaction with the course instructor would be not coming to office hours and communicating to the instructor. If the students do not tell the what is happening to their study, the teacher can not help the students. Without the instructor's guidance, students will not know if they are going on the right path or not. 
```

Comment:
Please make sure to add a description for all 5 assessment criteria mentioned in the beginning of the file:
```
None
Inadequate
Adequate
Good
Excellent
```